Mage/src/main/java/mage/abilities/common/DrawCardTriggeredAbility.java

```diff
@@ -90,17 +94,16 @@ public String generateTriggerPhrase() {
     }
 
     @Override
-    public DrawSecondCardTriggeredAbility copy() {
-        return new DrawSecondCardTriggeredAbility(this);
+    public DrawCardTriggeredAbility copy() {
+        return new DrawCardTriggeredAbility(this);
     }
 }
 
-class DrawSecondCardWatcher extends Watcher {
+class DrawCardWatcher extends Watcher {
```

Comment:
I don't think this watcher should be refactored like this, it's set up the way it is because tracking the second card drawn is more complicated than tracking e.g. the second spell cast or second land played

Reply:
What the refactor means is that tracking "Nth card drawn" should be the same independently of the value of N.
We don't want to have N different wathers.
If we do, please explain me the reason.

If you prefer, I can refactor this to "DrawNthCardWather", I just don't like complex Class names.